docs/01-overview/README.md

```diff
@@ -8,6 +8,12 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
 
 ![overview](./assets/kyma-overview.svg)
 
+## Modular Kyma
+
+Classic Kyma offers a fixed set of mandatory components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offered as independent modules.
+
+The modular approach boosts Kyma's extensibility and scalability and makes it cost-effective. With fully modularized Kyma, you can choose from many of its modules to facilitate your business needs best.  You can enable or disable modules on demand and configure them, thus determining the costs generated by the infrastructure that your modules use.
```

Comment:
Kyma is cost-effective, we are improving on that with modularization.  
Maybe even more importantly, we are reducing the attack surface area by not running components that are not needed. We also reduce the upgrade time and risk by not having to upgrade unnecessary components.